### ISSUE TITLE
Private repo conflicts with public repo of the same name

### ISSUE THREAD

**Issue by memelet** [raineorshine/npm-check-updates#611]
- [x] I have read the list of [known issues](https://github.com/tjunnone/npm-check-updates#known-issues) before filing this issue.
- [x] I have searched for [similiar issues](https://github.com/tjunnone/npm-check-updates/issues) before filing this issue.

* node version: 10.15.3
* npm version: 6.4.1
* npm-check-updates version: 3.2.1

---------------------------

This is not a npm-check-updates specific problem, as I have been able to reproduce with npm-check. I have one specific module in our private nexus repo that is never reported as out of date. Other modules in the private repo are detected correctly.

I have enabled `--loglevel silly` but don't get any output more than `--loglevel verbose`. Is there any other I can see the details of what is being checked?

**Comment by memelet** [comment]
Ok, I found the problem. The module we have in our private repo has the same name as a public repo in npmjs. The latest version returned by both npm-check-updates and npm-check are for the public repo, where the current version is for the nexus repo. 

What is strange is that `npm-check-updates --upgrade` will update the version to the version in the private repo.

**Comment by srmagura** [comment]
I tried to reproduce this and it worked correctly, so this can be closed, assuming I followed the correct steps. 

Here's what I did: 
[Link to repo](https://github.com/srmagura/npm-check-updates-611)

1. Set up a local private npm registry using Verdaccio.
2. Published an empty package to my private registry. The package name is `react` and the version is 1.0.0.
3. Created a new package called `my-application` and set it to use my private registry by adding `registry=http://localhost:4873` to `my-application/.npmrc`.
4. In `my-application`, `npm install react`. It added `react@^1.0.0` to `package.json` and put the empty `react` package in `my-application/node_modules`.
5. Ran `npm-check-updates` in `my-application`. It printed "All dependencies match the latest package versions :)", which is the expected result. If it had looked up `react` on the public npm registry, it would have said my dependency was out of date since the current version of the real `react` package is 18.2.0.
